drive/client.bal

```diff
@@ -83,7 +83,7 @@ public client class Client {
     # 
     # + filterString - The filter string
     # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
-    @display {label: "Get all files"}
+    @display {label: "Get All Files"}
     remote isolated function getAllFiles(@display {label: "Filter String"} string? filterString = ()) 
```

Comment:
Shall we merge the two operations  `getAllFiles ` and `filterFiles `. We can keep `getAllFiles` with orderBy as an optional. In Bal Doc specify we can receive all files filtered and optionally ordered.

Reply:
Hi,
Did the changes. 
Thanks.